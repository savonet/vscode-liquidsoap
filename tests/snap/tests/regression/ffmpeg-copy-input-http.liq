s = input.http("foo")
output.srt(port=10001, %ffmpeg(%audio.copy), fallible=true, s)

def f() =
  test.pass()
end

test.check(f)
