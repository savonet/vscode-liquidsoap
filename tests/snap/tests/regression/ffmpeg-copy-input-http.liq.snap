>s = input.http("foo")
#^^^^^^^^^^^^^^^ source.liquidsoap
#               ^ source.liquidsoap string.quoted.double.liquidsoap
#                ^^^ source.liquidsoap string.quoted.double.liquidsoap
#                   ^ source.liquidsoap string.quoted.double.liquidsoap
#                    ^^ source.liquidsoap
>output.srt(port=10001, %ffmpeg(%audio.copy), fallible=true, s)
#^^^^^^^^^^^^^^^^ source.liquidsoap
#                ^^^^^ source.liquidsoap constant.numeric.integer
#                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.liquidsoap
>
>def f() =
#^^^ source.liquidsoap keyword.control.liquidsoap
#   ^^^^^^^ source.liquidsoap
>  test.pass()
#^^^^^^^^^^^^^^ source.liquidsoap
>end
#^^^ source.liquidsoap keyword.control.liquidsoap
>
>test.check(f)
#^^^^^^^^^^^^^^ source.liquidsoap
>